packages/dappmanager/src/calls/getHostUptime.ts

```diff
@@ -0,0 +1,14 @@
+import {shellHost, shell} from "@dappnode/utils";
+
+/**
+ * Returns uptime of the host
+ */
+export async function getHostUptime(): Promise<string> {
+    try{
+        const output = await shellHost("uptime -- -p");
+        return output;
+    }catch(error){
+        console.log("Error getting uptime of the host");
+        throw error;
```

Comment:
If you want to throw the error, this error will be shown in the UI so you could format it a bit by appending a custom message, i.e:
```js
error.message += `Unable to retrieve the uptime from the host machine: ${error.message}`
throw error
```